hooks/useNotification.tsx

```diff
@@ -0,0 +1,25 @@
+import { useRecoilState } from 'recoil'
+import { notificationState } from 'lib/recoil-store/atoms'
+import { INotification } from 'lib/interfaces'
+import { useEffect } from 'react'
+
+const useNotification = () => {
+    const [notification, setNotification] = useRecoilState<INotification>(notificationState)
```

Comment:
The notification hook should allow multiple notifications to be displayed at the same time, it should be `notifications` of type `INotification[]`